README.md

```diff
@@ -14,7 +14,7 @@
 by [@colinhacks](https://twitter.com/colinhacks)
 </p>
 
-> Hi! Colin here, creator of Zod. I hope you find it easy to use and powerful enough for all your use cases. If you have any issues or suggestions, please  [open an issue](https://github.com/colinhacks/zod/issues/new)! 
+> Hi! Colin here, creator of Zod. I hope you find it easy to use and powerful enough for all your use cases. If you have any issues or suggestions, please [open an issue](https://github.com/colinhacks/zod/issues/new)!
```

Comment:
Not sure if this should be reverted, it seems like those extra spaces shouldn't be there anyway.